[full-ci][tests-only]Add support to stop and start service in ociswrapper (#10776)
owncloud/ocis: tests/ociswrapper/README.md

    - `200 OK` - oCIS server is stopped
    - `500 Internal Server Error` - Unable to stop oCIS server

6. `POST /services/{service-name}`

>     Restart oCIS with service excluded and start excluded oCIS service individually, not covered by the oCIS supervisor.

Reviewer: ```suggestion
    Restart oCIS instances without specified service and start that service independently
```
If I understand the wording correctly, this feels right

    - `200 OK` - oCIS server is stopped
    - `500 Internal Server Error` - Unable to stop oCIS server

   6. `POST /services/{service-name}`

       Restart oCIS with service excluded and start excluded oCIS service individually, not covered by the oCIS supervisor.

       Body of the request should be a JSON object with the following structure:

       ```json
       {
         "ENV_KEY1": "value1",
         "ENV_KEY2": "value2"
       }
       ```

      > **⚠️ WARNING:**
      > You need to set the address of the debug server to expose the port in CI

>       > `{SERVICE-NAME}_DEBUG_ADDR`

Reviewer 2: add it as a note.

>       > You need to set the address of the debug server to expose the port in CI
>       > `{SERVICE-NAME}_DEBUG_ADDR`

Reviewer 2: ```suggestion
      > You need to set the proper addresses to access the service from other steps in the CI pipeline.
      > `{SERVICE-NAME}_DEBUG_ADDR=0.0.0.0:{DEBUG_PORT}`
      > `{SERVICE-NAME}_HTTP_ADDR=0.0.0.0:{HTTP_PORT}`
```
